api-reference/beta/api/profilephoto_get.md

```diff
@@ -4,28 +4,27 @@
 
 Get the specified [profilePhoto](../resources/profilephoto.md) or its metadata (**profilePhoto** properties).
 
-A GET photo operation first looks for the specified photo in the user's mailbox on Exchange Online, and if 
-it's not available there, then looks in Azure Active Directory (AAD).
+A GET photo operation first attempt sto retrieve the specified photo from Office 365. If the photo is not available in Office 365, the API attempts to retrieve the photo from Azure Active Directory (AAD).
 
-The supported sizes of HD photos on Exchange Online are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
-'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in AAD.
+The supported sizes of HD photos on Office 365 are as follows: '48x48', '64x64', '96x96', '120x120', '240x240', 
+'360x360','432x432', '504x504', and '648x648'. Photos can be any dimension if they are stored in Azure Active Directory.
```

Comment:
I think you should ask for some help from Kostas (Konstantin Seleskerov <koselesk@microsoft.com>).  He knows much better how this API works with regards to Office.  AAD stores just one photo, and doesn't do any rescaling/resizing when reading it back.  I don't know what the photos (different to the photo) API does if the photo is ONLY in AAD.

Reply:
The statement above regarding sizes of the photo is correct.